contracts/test/TestAggregatorV3.sol

```diff
@@ -84,4 +86,8 @@ contract TestAggregatorV3 is AggregatorV3Interface {
             roundData[latestRound].answeredInRound
         );
     }
+
+    function computeRoundId(uint16 phaseId, uint64 aggregatorRoundId) external pure returns (uint80) {
```

Comment:
Add a comment to help understand why we put it here (just for knowing how `roundId` works)!